src/main/java/mate/academy/lib/Injector.java

```diff
@@ -33,6 +33,16 @@ public static Injector getInstance(String mainPackageName) {
         return injector;
```

Comment:
Are you sure that you should change something in this file?

Reply:
> Are you sure that you should change something in this file?

I didn't want, mvn clean package fails because all overloaded methods should be next to each other.